### ISSUE TITLE
Suggestions of @mentions and #hashtags go back to what we wrote when typing backspace

### ISSUE THREAD

**Issue by Porrumentzio** [tuskyapp/Tusky#1850]
When starting to write a @mention, let's say our aim is to mention @Anamorfo3, and we write `@Anam`, there appear suggestion. When **choosing** those **suggestion**, it **completes** the name and **add** an **space** after.
Happens the same with #hastags.

The **problem** comes when, for removing that space, we type **backspace**: all the **suggestion** goes **back** to those letters we wrote.
This is really tedious as on some languages we need to write declinations just after the name.

The expected behaviour would be to treat that suggestion as normal text, which can be deleted letter by letter.

* * * *
- Tusky Version: 11.0
- Android Version: 7.0
- Android Device: Vernee Thor E
- Mastodon instance (if applicable): mastodon.eus and esperanto.masto.host

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by realpixelcode** [comment]
It also applies to adding braces, e.g.: “Hi, I'm new here (#introductions)!”

**Comment by connyduck** [comment]
I can not reproduce that on my Fairphone 4 with Tusky 19. Can you maybe share a video and tell me what device, keyboard, Android & Tusky versions you are using? @realpixelcode

**Comment by realpixelcode** [comment]
@connyduck I just tried again and couldn't reproduce neither of these issues. (Possibly, I confused this with the Mastodon app. 😅)

**Comment by connyduck** [comment]
Still can't reproduce with Tusky 24 on my Fairphone 4 but can with an Android 9 emulator. Thanks for bringing this up again.

**Comment by Porrumentzio** [comment]
Yes sorry. I edited my previous post to add that the problem seems to be caused by AnySoftKeyboard, as using Tusky with AOSP keyboard works fine.
You can close this issue, I reported it on AnySoftKeyboard's repo
